Packages/com.chisel.components/Chisel/Components/API.public/Components/Base/CSGNode.cs

```diff
@@ -411,9 +380,6 @@ internal virtual void AddPivotOffset(Vector3 worldSpaceDelta)
         }
 
 #if UNITY_EDITOR
-            // The icon used in the hierarchy
-        public abstract GUIContent Icon { get; }
```

Comment:
Before Icons where specified in a UNITY_EDITOR part of the code in Components, this functionality is now moved to the same files as the component editors.